modules/fastcv/src/moments.cpp

```diff
@@ -20,36 +20,30 @@ cv::Moments moments(InputArray _src, bool binary)
     Mat src = _src.getMat();
 
     cv::Moments m;
-	if( size.width == 0 || size.height == 0 )
-        return m;
-
-	fcvMoments* mFCV = new fcvMoments();
+    fcvMoments mFCV;
```

Comment:
Since `mFCV` is not a pointer now, its usage should be changed:
* no need to call `delete mFCV`
* should be dereferenced as `mFCV.m00`, not `mFCV->m00`

Reply:
these changes should be make in next PR, revert back to the original